manifests/pcidevices-operator.yaml

```diff
@@ -71,7 +74,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-          image: rancher/harvester-pcidevices:master-head
+          image: tlehman/pcidevices:dev-kubevirt
```

Comment:
This was only for testing, anyone who is reviewing this who wants to test it should use this image, then I'll switch it back when this PR is merged.